rasa/core/channels/console.py

```diff
@@ -50,14 +58,20 @@ def print_bot_output(message, color=rasa.cli.utils.bcolors.OKBLUE):
         )
 
 
-def get_cmd_input():
-    response = questionary.text(
-        "", qmark="Your input ->", style=Style([("qmark", "#b373d6"), ("", "#b373d6")])
-    ).ask()
+def get_cmd_input(button_question: questionary.Question):
+    if button_question:
```

Comment:
Can we make it more expressive? I would initialize `button_question` with `None` and also return `None` in `print_bot_output` at the end. The check here I would modify to `button_question is not None`.